packages/wallet/src/redux/protocols/close-ledger-channel/states.ts

```diff
@@ -0,0 +1,71 @@
+import { WithdrawalState } from '../withdrawing/states';
+import { AdvanceChannelState } from '../advance-channel';
+import { StateConstructor } from '../../utils';
+import { ProtocolState } from '..';
+
+export interface WaitForWithdrawal {
+  type: 'CloseLedgerChannel.WaitForWithdrawal';
+  processId: string;
+  withdrawal: WithdrawalState;
+  channelId;
+}
+export interface WaitForConclude {
+  type: 'CloseLedgerChannel.WaitForConclude';
+  processId: string;
+  concluding: AdvanceChannelState;
+  channelId;
+}
+
+export interface Failure {
+  type: 'CloseLedgerChannel.Failure';
+  reason: string;
+}
+
+export interface Success {
+  type: 'CloseLedgerChannel.Success';
+}
+
+// -------
+// Constructors
+// -------
+
+export const waitForWithdrawal: StateConstructor<WaitForWithdrawal> = p => {
```

Comment:
Are we committed to disallowing single letter variables? (https://github.com/magmo/apps/issues/559)

I feel like it can be pretty readable (and is something we do all over the place)